smartsim/_core/utils/helpers.py

```diff
@@ -284,6 +287,16 @@ def execute_platform_cmd(cmd: str) -> t.Tuple[str, int]:
     return process.stdout.decode("utf-8"), process.returncode
 
 
+def _stringify_id(_id: int) -> str:
+    """Return the CPU id as a string if an int, otherwise raise a ValueError"""
```

Comment:
Can we add param, returns, and raises doc strings to this?

Reply:
added